packages/client/hmi-client/src/workflow/ops/simulate-julia/tera-simulate-julia.vue

```diff
@@ -203,15 +202,21 @@ const updateState = () => {
 	emit('update-state', state);
 };
 
-const runSimulate = async () => {
-	const modelConfigurationList = props.node.inputs[0].value;
-	if (!modelConfigurationList?.length) return;
+// Main entry point
+const run = async () => {
+	showSpinner.value = true;
+	const simulationId = await makeForecastRequest();
+	await pollResult(simulationId);
+	await processResult(simulationId);
+	await lazyLoadSimulationData(simulationId);
```

Comment:
Does those steps needs to be in order? If yes, could we display multiple state of loading to let the user know what is happening?
Like: "running simulation", "processing simulation results" and "loading simulation results "

Reply:
They need to be in order, but for sciml simulate, it happens too fast that progress messages will look like flickering bugs.